Jenkinsfile_CNP

```diff
@@ -8,10 +8,27 @@ def component = "consumer"
 
 def channel = '#rpe-build-notices'
 
+List<LinkedHashMap<String, Object>> secrets = [
+  secret('test-s2s-name', 'TEST_S2S_NAME'),
+  secret('test-s2s-secret', 'TEST_S2S_SECRET'),
+  secret('s2s-url', 'S2S_URL'),
+  secret('send-letter-producer-url', 'SEND_LETTER_PRODUCER_URL')
```

Comment:
Tests will need to use the producer to put messages into the queue. Can't do it directly, as those letters will not exist in the DB, which, in turn, will lead to errors on updating their status.